components/lib/messages/messages.d.ts

```diff
@@ -22,6 +22,9 @@ export declare type MessagesPassThroughTransitionType = ReactCSSTransitionProps
  */
 export interface MessagesPassThroughMethodOptions {
     props: MessagesProps;
+    state: MessagesState;
+    hostName?: string;
```

Comment:
I will have to look. What is hostname?  Weird property for messages.

Reply:
Since last release I see that prop in some components... I don't know what it is neither. If you see my PR #6902 in the stackblitz reproducer, you will see that prop also for bodyRow in Datatable component.
I only include that prop here because it is passed through MessagesPassThroughMethodOptions.